ehrapy/api/encode/__init__.py

```diff
@@ -18,23 +23,23 @@ def encode(ann_data: AnnData, autodetect: bool = False, encodings: Dict[str, Lis
         3. count encoding (https://contrib.scikit-learn.org/category_encoders/count.html)
 
     Args:
-        ann_data: The inital AnnData object
+        data: The initial AnnData or MuData object
         autodetect: Autodetection of categorical values
-        encodings: Only needed if autodetect set to False.
-        A dict containing the categorical name and the encoding mode for the respective column.
+        encodings: Only needed if autodetect set to False (or False for some columns in case of a MuData object).
+        A dict containing the encoding mode and categorical name for the respective column (for each AnnData object in case of MuData object).
 
     Returns:
-        An :class:`~anndata.AnnData` object with the encoded values in X
+        An :class:`~anndata.AnnData` object with the encoded values in X or None (in case of MuData object)
 
     Example:
         .. code-block:: python
 
             import ehrapy.api as ep
             adata = ep.io.read(...)
             # encode col1 and col2 using label encoding and encode col3 using one hot encoding
-            ep.encode.encode(adata, autodetect=False, {'label_encoding': ['col1', 'col2'], 'one_hot_encoding': ['col3']})
+            adata_encoded = ep.encode.encode(adata, autodetect=False, {'label_encoding': ['col1', 'col2'], 'one_hot_encoding': ['col3']})
     """
-    return Encoder.encode(ann_data, autodetect, encodings)
+    return Encoder.encode(data, autodetect, encodings)
```

Comment:
```suggestion
    return Encoder.encode(adata, autodetect, encodings)
```

Reply:
Its not always an AnnData object, but also sometimes a MuData object